channels/ar.m3u

```diff
@@ -89,3 +89,9 @@ https://wowza.telpin.com.ar:1935/telpintv/smil:ttv.stream.smil/playlist.m3u8
 http://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TVManaArgentina.ar" tvg-name="TV Maná Argentina" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://i.imgur.com/GVI8kkp.jpg" group-title="Local",TV Maná Argentina (576p)
 http://streamspub.manasat.com:1935/tvar/tvmanaar2/playlist.m3u8
+#EXTINF:0,TVP
+https://cntlnk-main-edge-access.secure.footprint.net/b16/ngrp:c7_vivo01_dai_source-20001_all/c7_vivo01_dai_source-20001_all/playlist.m3u8
+#EXTINF:0,TVP (480p)
+https://cntlnk-main-edge-access.secure.footprint.net/b16/ngrp:c7_vivo01_dai_source-20001_all/c7_vivo01_dai_source-20001_480p.m3u8
+#EXTINF:0,El Nueve
+https://ar-elnueve-elnueve-live.ned.media/live.m3u8?iut=eyJwYXJhbXMiOnsiZXhwIjoxNjIzMTYxNzg0LCJzZXNzaW9uIjoiOTIuMTE5LjE3Ny44OSIsIndoaXRlbGlzdCI6WyI5Mi4xMTkuMTc3Ljg5Il19LCJzaWduYXR1cmUiOiI4MjQxOTNiYjk4ZGEyOTRiZjdlYmJkNGVkODhlZjZmOGM3ZjQ0MzNkIn0=
```

Comment:
This link also no longer works:
![image](https://user-images.githubusercontent.com/7253922/121530520-2ee05380-ca06-11eb-85c9-ffafe44cd3bf.png)